libs/json-model-graph/src/lib/services/graph-form-context.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Inject, Injectable, Optional, SkipSelf } from '@angular/core';
+import { DataModelDefinitionHelper, DATA_MODEL_DEFINITION_HELPER_TOKEN } from '@cognizone/json-model';
+
+import { GraphWrapper } from './graph-wrapper';
+import { GraphWrapperFactory } from './graph-wrapper.factory';
+
+@Injectable()
+export class GraphFormContextService {
```

Comment:
This might not be the best name now that I think of it. `Context` is a broad keyword, but I'm afraid it might make people believe that this has anything to do with `TypedResourceContext` (which it doesn't 😅 )

Reply:
Could be just `GraphFormService` maybe?